framework/doc/content/syntax/Mesh/index.md

```diff
@@ -23,6 +23,24 @@ of Mesh objects refer to the individual object pages listed below.
 
 !syntax list /Mesh objects=False actions=True subsystems=False
 
+## MeshGenerator System
+
+The MeshGenerator System is useful for programmatically constructing a mesh. This includes generating the mesh
+from a serious of points and connectivity, adding features on the fly, linearly transforming the mesh, stitching
+together pieces of meshes, etc. There are several built-in generators but this system is also extendable. MeshGenerators
+may or may not consumer the output from other generators and produce a single mesh. They can be chained together
+through dependencies so that complex meshes may be built up from a series of simple processes.
+
+### DAG and final mesh selection
+
+When chaining together several MeshGenerators, you are implicitly creating a DAG (directed acyclic graph).
+MOOSE evaluates and generates the individual objects to build up your final mesh. If your input file has
+multiple end points, (e.g. B->A and C->A) then MOOSE will issue an error and terminate. Generally, it doesn't m
```

Comment:
Extra m at the end of the line hiding here

Reply:
Dammnit!